openslides_backend/presenter/get_users.py

```diff
@@ -0,0 +1,133 @@
+from operator import itemgetter
+from typing import Any, Dict, List
+
+import fastjsonschema
+
+from ..services.datastore.deleted_models_behaviour import DeletedModelsBehaviour
+from ..shared.exceptions import PresenterException
+from ..shared.patterns import Collection
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+get_users_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "title": "get_users",
+        "description": "get user ids data",
+        "properties": {
+            "start_index": {"type": "integer"},
+            "entries": {"type": "integer"},
+            "sort_criteria": {
+                "type": "array",
+                "item": {
+                    "type": "string",
+                    "enum": ["username", "first_name", "last_name"],
+                },
+            },
+            "reverse": {"type": "boolean"},
+            "include_temporary": {"type": "boolean"},
+            "filter": {"type": ["string", "null"]},
+        },
+        "required": [],
+        "additionalProperties": False,
+    }
+)
+
+ALLOWED = {
```

Comment:
Please add a comment that the values are default values for `None`.

Reply:
Done.